routes/web.php

```diff
@@ -88,4 +88,15 @@
     Route::post('/membership/reject-details/{user}', 'MembershipController@rejectDetails')->name('membership.reject');
     Route::get('/membership/update-details/{user}', 'MembershipController@editDetails')->name('membership.edit');
     Route::put('/membership/update-details/{user}', 'MembershipController@updateDetails')->name('membership.update');
+
+    // Members Projects and DNH labels
+    Route::get('projects/{project}/markActive', 'Members\ProjectController@markActive')->name('projects.markActive');
+    Route::get('projects/{project}/markAbandoned', 'Members\ProjectController@markAbandoned')->name('projects.markAbandoned');
+    Route::get('projects/{project}/markComplete', 'Members\ProjectController@markComplete')->name('projects.markComplete');
```

Comment:
These are making changes to a resource, so should be POST, or PATCH, at a minimum.

Even better would be removing these entirely and achieving this with a PATCH on the /projects/{project} resource updating the `state` field

Reply:
switched to patch, but keeping the routes